README.md

```diff
@@ -4,40 +4,124 @@
+- **Test run**: The trainer has a `test_run` function to train the model for few iterations and test if
+    - the model is executable (burn test),
+    - the validation is deterministic/reproducible, and
+    - the model changes the parameter during training.
+- **Virtual minibatch**:
+  - The `Trainer` usually does not know if the model is trained with a single example or multiple examples (minibatch), because the examples that are yielded from the dataset are directly forwarded to the model.
+  - When the `virtual_minibatch_size` option is larger than one, the trainer calls the forward and backward step `virtual_minibatch_size` times before applying the gradients. This increases the number of examples over which the gradient is calculated while the memory consumption stays similar. See [here](doc/virtual_batch_size_multi_gpu.md) for a more thorough explanation.
+- **Multi-GPU training**: Easily deploy your model onto multiple GPUs to increase the total batch size and speed up the training. See [here](doc/virtual_batch_size_multi_gpu.md#L68) for implementation details and the [example](padertorch/contrib/examples/multi_gpu) for how to enable it.
+- **Support for lazy data preparation**: Ever tired of pre-computing features, taking up time and space on the hard disk? Padertorch works with lazy dataloaders (e.g., [lazy_dataset](https://github.com/fgnt/lazy_dataset)) which extract the features on the fly!
+
+# Features for Application in Deep Learning
+
+Padertorch provides a selection of frequently used network architectures and functionalities such as activation and normalization, ready for you to integrate into your own models.
+
+- [Multi-layer Feed-Forward](padertorch/modules/fully_connected.py): Multiple fully-connected layers with non-linearity and dropout.
+- [CNN](padertorch/contrib/je/modules/conv.py) (currently subject to breaking changes and hardly any documentation): 1d- and 2d-CNNs with residual connections, dropout, gated activations, batch and sequence norm and correct handling of down- and upsampling.
+- [Normalization](padertorch/modules/normalization.py): Perform normalization of arbitrary axes/dimensions of your features, keep track of running statistics and apply learnable affine transformation.
+- [Collection of activation functions](padertorch/ops/mappings.py): Fast access of various activation functions with just a string.
+- [Losses](padertorch/ops/losses): We provide an implementation of the [Kullback-Leibler divergence](padertorch/ops/losses/kl_divergence.py) and different [regression](padertorch/ops/losses/regression.py) objectives.
+
+## Advanced Architectures
+
+- [Dual-Path RNN (DPRNN)](padertorch/modules/dual_path_rnn.py): See the [paper](https://arxiv.org/abs/1910.06379).
```

Comment:
Perhaps mention wavenet here as well? I think it is at least as prominent

Reply:
Good idea. I am unsure whether to add it here or below the next section since the Wavenet is more relevant for speech.
I am also thinking of removing this section and putting the DPRNN below the next section, too.

Reply:
I think we can move all of this further down, since these are benefits but not the main selling points.